File: A/71A.py
n = int(input())

L = []

for i in range(n):
    str1 = str(input())
    L.append (str1)

for i in L:
    if (len(i) > 10):
        print(i[0] + str(len(i)-2) + i[-1])
    else:
        print(i)

File: A/71A.cpp
#include <bits/stdc++>

int main() {
    int n,len;
    std::cin >> n;

    std::string str1[n];
    for (int i=0; i <n ; i++) 
        std::cin >> str1[i];

    for (int i=0; i <n ; i++) {
        len = str1[i].length();
        if (len >= 10)
            std::cout << str1[i][0] << to_string(len) << str1[i][-1]; 
    }

    return 0;
}
